### PR TITLE
Fix dhcpd startup

### PR DESCRIPTION
**Pull request by pprindeville**
Maintainer: me / @Naoir
Compile tested: x86_64, generic, HEAD (5f5d812)
Run tested: installed on test router (APU2), and rebooted. DHCP seems fine.

Description:

### CODE REVIEW

**Comment by neheb** [comment]
Requires a PKG_RELEASE bump.

**Comment by pprindeville** [comment]
> Requires a PKG_RELEASE bump.

Done.

File: net/isc-dhcp/files/dhcpd.init [hunk 1]
```diff
@@ -302,13 +309,33 @@ start_service() {
...
+
+	if [ -n "$trigger" -a $ignore -eq 0 ] ; then
+		procd_add_reload_interface_trigger "$trigger"
+	fi
 }
+
+service_triggers() {
+	if [ -n "$DHCPD_BOOT" ] ; then
+		# Make the first start robust to slow interfaces; wait a while
+		procd_add_raw_trigger "interface.*.up" 5000 /etc/init.d/dhcpd restart
```
Comment (EricLuehrsen) [inline net/isc-dhcp/files/dhcpd.init:332]: I have struggled with these trigger lines on other packages. DHCPv6 ISP can generate a lot of IFUP like events when broadcasting minor link value updates. You seem to boot-start DHCP processing after the first interface actually is online, then only re-trigger on DHCP enabled interfaces exclusively. This should work but I wonder how to certainly test that a startup is not stalled and that the daemon isn't purged/reloaded to frequently.

Comment (pprindeville) [reply net/isc-dhcp/files/dhcpd.init:332]: We could use Qemu and OpenvSwitch to come up with a test harness...

**Pull request merged**